### DISCUSSION TITLE
gatsby-source-wordpress - Dev's recommended way to trigger rebuild by team?

### DISCUSSION THREAD

**Discussion by sol-idsnake** [gatsbyjs/gatsby#29568] [Help]
I am fairly new to Gatsby and am looking to implement a prototype before using Gatsby in production with Wordpress. Huge shoutout to the team for all the progress and awesome features.

I work alongside a team, and after the they implement simple changes like featured images, content edits, etc., I am looking for the ideal solution to trigger a rebuild. I don't think it would be feasible for a Dev to manually trigger a rebuild every time a change is made.

I've researched writing a small plugin to trigger this manually, but are there any resources or information around this topic already ?

Thanks for any input.

**Comment by sol-idsnake** [comment]
Would still like to hear about how others solve this issue of independence on their team.

**Comment by KyleAMathews** [comment]
Have you checked out https://wordpress.org/plugins/wp-gatsby/? It handles Preview/Publish automatically along with Gatsby Cloud

**Reply by sol-idsnake** [reply]
@KyleAMathews that's a great start, I hadn't seen the docs for it. Thanks a lot
